deployments/infrastructure-target-aws.nix

```diff
@@ -29,17 +30,46 @@ rec {
       ./../modules/amazon-base.nix
     ];
 
-    deployment.keys.tarsnap = {
-      keyFile = ./../static/tarsnap-cardano-deployer.secret;
-      destDir = "/var/lib/keys";
-    };
-
     deployment.ec2 = {
       inherit accessKeyId;
-
       instanceType = mkForce "r3.2xlarge";
       ebsInitialRootDiskSize = mkForce 50;
       associatePublicIpAddress = true;
+      securityGroups = [
+        resources.ec2SecurityGroups."allow-all-ssh-${region}-${org}"
+      ];
+    };
+  };
+
+  resources = {
+    ec2SecurityGroups = {
+      "allow-deployer-ssh-${region}-${org}" = {
+        inherit region accessKeyId;
+        description = "SSH";
+        rules = [{
+          protocol = "tcp"; # TCP
+          fromPort = 22; toPort = 22;
+          sourceIp = deployerIP + "/32";
+        }];
+      };
+      "allow-all-ssh-${region}-${org}" = {
+        inherit region accessKeyId;
+        description = "SSH";
+        rules = [{
+          protocol = "tcp"; # TCP
+          fromPort = 22; toPort = 22;
+          sourceIp = "0.0.0.0/0";
+        }];
+      };
+      "allow-public-www-${region}-${org}" = {
+        inherit region accessKeyId;
+        description = "WWW";
+        rules = [{
+          protocol = "tcp"; # TCP
+          fromPort = 443; toPort = 443;
+          sourceIp = "0.0.0.0/0";
+        }];
+      };
```

Comment:
№3

We probably want to have the more centralised configuration approach for security groups applied to infrastructure as well.